client/src/main/java/io/pravega/client/stream/AckFuture.java

```diff
@@ -1,25 +0,0 @@
-/**
- * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- */
-package io.pravega.client.stream;
-
-import java.util.concurrent.Executor;
-import java.util.concurrent.Future;
-
-public interface AckFuture extends Future<Void> {
```

Comment:
Earlier since there was no internal thread pool in the EventStreamWriter the AckFuture mechanism was used to ensure that a `flushInternal `was invoked when a AckFuture.get() was invoked by the application. This is now redundant since the threadpool now ensures that SegmentSealed is handled in the threadpool instead of the application's thread.